src/contributing/managing-the-playbook.md

```diff
@@ -4,6 +4,8 @@ permalink: /:path/:basename/
 redirect_from:
   - /guides/managing
 ---
+**Needs review**
```

Comment:
Is there a Trello card or an issue for this? It probably needs someone to coordinate a discussion?

Reply:
Nope. Please do start that conversation. As I said above, I'm not trying to tackle any of the content here.